src/Gui/TaskView/TaskSolverMessages.cpp

```diff
@@ -0,0 +1,138 @@
+// SPDX - License - Identifier: LGPL - 2.1 - or -later
+/****************************************************************************
+ *                                                                          *
+ *   Copyright (c) 2011 Jürgen Riegel <juergen.riegel@web.de>              *
+ *                                                                          *
+ *   This file is part of FreeCAD.                                          *
+ *                                                                          *
+ *   FreeCAD is free software: you can redistribute it and/or modify it     *
+ *   under the terms of the GNU Lesser General Public License as            *
+ *   published by the Free Software Foundation, either version 2.1 of the   *
+ *   License, or (at your option) any later version.                        *
+ *                                                                          *
+ *   FreeCAD is distributed in the hope that it will be useful, but         *
+ *   WITHOUT ANY WARRANTY; without even the implied warranty of             *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU       *
+ *   Lesser General Public License for more details.                        *
+ *                                                                          *
+ *   You should have received a copy of the GNU Lesser General Public       *
+ *   License along with FreeCAD. If not, see                                *
+ *   <https://www.gnu.org/licenses/>.                                       *
+ *                                                                          *
+ ***************************************************************************/
+
+#include "PreCompiled.h"
+#ifndef _PreComp_
+#include <QAction>
+#include <QMenu>
+#endif
+
+#include <Gui/Application.h>
+#include <Gui/BitmapFactory.h>
+#include <Gui/Command.h>
+#include <Gui/ViewProvider.h>
+
+#include "TaskSolverMessages.h"
+#include "ui_TaskSolverMessages.h"
+
+
+using namespace Gui;
+using namespace Gui::TaskView;
+namespace sp = std::placeholders;
+
+TaskSolverMessages::TaskSolverMessages(const QPixmap& icon,
+                                       const QString& title)
+    : TaskBox(icon, title, true, nullptr)
+    , ui(new Ui_TaskSolverMessages)
+{
+    // we need a separate container widget to add all controls to
+    proxy = new QWidget(this);
```

Comment:
linter is right, but do we even need to keep the proxy as member of the TaskSolverMessages? I'd just move it here as `auto* proxy = new QWidget(this)`